ecs.tf

```diff
@@ -5,12 +5,13 @@ data "aws_ecs_cluster" "target_cluster" {
 resource "aws_ecs_task_definition" "main_task" {
   family                   = "${var.app_name}-tsk"
   requires_compatibilities = [var.launch_type]
-  network_mode             = "bridge"
+  network_mode             = var.network_mode
```

Comment:
If this has to be set to a certain value for fargate to work we could make this a bit more automatic. You could put a conditional here for the launch type and just set it to `awsvpc` if we're doing fargate.

Reply:
Updated